util/network.go

```diff
@@ -15,16 +15,26 @@ import (
 // CreateListener crates socket listener based on DSN definition.
 func CreateListener(address string) (net.Listener, error) {
 	dsn := strings.Split(address, "://")
-	if len(dsn) != 2 {
-		return nil, errors.New("invalid DSN (tcp://:6001, unix://file.sock)")
-	}
 
-	if dsn[0] != "unix" && dsn[0] != "tcp" {
-		return nil, errors.New("invalid Protocol (tcp://:6001, unix://file.sock)")
+	var protocol string
+	var param    string
```

Comment:
Also, to be a little bit strict, this is not a `protocol` and `param` in terms of `net.Listen` function. This is the `network` and `address`.

Reply:
Addressed in b327815.